src/index.html

```diff
@@ -23,32 +23,18 @@ <h1>Leaderboard</h1>
                             <h2>Recent Scores</h2>
                         </div>
                         <div class="col-md-4">
-                            <a href="" class="btn btn-primary">Refresh</a>
+                            <a href="" class="btn btn-primary" id="showData">Refresh</a>
                         </div>
                     </div>
                     <table class="table table-striped">
-                        <tbody>
-                            <tr>
-                                <td>Name : 100</td>
-                            </tr>
-                             <tr>
-                                <td>Name : 100</td>
-                            </tr>
-                             <tr>
-                                <td>Name : 100</td>
-                            </tr>
-                             <tr>
-                                <td>Name : 100</td>
-                            </tr>
-                             <tr>
-                                <td>Name : 100</td>
-                            </tr>
+                        <tbody class="tbody">
+                            
                         </tbody>
                     </table>
                 </div>
-                <div class="col-md-6">
+                <div class="col-md-6 score-box">
                     <h2>Add your score</h2>
-                    <form action="#" method="post">
+                    <form action="#" method="post" id="score-form">
                         <input type="text" class="form-control" name="name" id="name" placeholder="Your name">
                         <input type="number" class="form-control mt-3" name="score" id="score" placeholder="Your score">
                         <button type="submit" class="btn btn-primary mt-3 d-flex justify-content-right">Submit</button>
```

Comment:
![image](https://github.com/Mahdi-Niazi/leaderboard-project/assets/93039359/738ae5c1-ec68-4990-9747-f2ec5686cb73)

**[OPTIONAL]**

- [ ] It would be nice if your app would clear the input fields after inserting the data into the API to make your app more professional by giving a great user experience 👍